kolibri/core/assets/src/utils/setupAndLoadFonts.js

```diff
@@ -91,10 +85,11 @@ function addFontStylesheetLink(href) {
 }
 
 export default function setupAndLoadFonts() {
-  if (passesRequirements(browser, modernFontBrowsers)) {
+  if (passesRequirements(browser, fontSwapSupportingBrowsers)) {
     addFontStylesheetLink(plugin_data.fullCSSFileModern);
+    loadFullFontsImmediately();
```

Comment:
Could return early here and avoid the `else` clause.

Reply:
yes! sounds like a comment I might usually leave :)